### PR TITLE
DDT-1091: OperationsEvent Controller

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

**Review by FinnAtAsseco — CHANGES_REQUESTED** [review]

File: jit-service/src/main/java/org/dcsa/jit/service/OperationsEventService.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class OperationsEventService {
+
+  private final OperationsEventRepository eventRepository;
+
+  public List<OperationsEvent> findAll() {
+    return eventRepository.findAll();
```
[the remark this answers is not in the document]
Comment (mp-assco) [reply jit-service/src/main/java/org/dcsa/jit/service/OperationsEventService.java]: Changed to TO instead.

File: jit-transfer-obj/src/main/java/org/dcsa/jit/transferobjects/PartyTO.java [hunk 2]
```diff
@@ -7,7 +7,7 @@
 import java.util.UUID;
 
 public record PartyTO(
-  UUID id,
+  @Size(max = 100) String id,
```
Comment (nt-as) [inline jit-transfer-obj/src/main/java/org/dcsa/jit/transferobjects/PartyTO.java:10]: I don't think we expose the Party ID in the JSON?

Comment (mp-assco) [reply jit-transfer-obj/src/main/java/org/dcsa/jit/transferobjects/PartyTO.java:10]: Not according to the swagger, no.

Comment (nt-as) [reply jit-transfer-obj/src/main/java/org/dcsa/jit/transferobjects/PartyTO.java:10]: ok, then I think this should be removed from the `PartyTO`

Comment (mp-assco) [reply jit-transfer-obj/src/main/java/org/dcsa/jit/transferobjects/PartyTO.java]: Sounds good to me. Removed!

**Review by nt-as — DISMISSED** [review]

**Review by nt-as — APPROVED** [review]

**Pull request merged**